tests/mock_tables/dbconnector.py

```diff
@@ -50,7 +64,16 @@ def __init__(self, *args, **kwargs):
             raise ValueError("Invalid db")
         self.pubsub = MockPubSub()
 
-        fname = os.path.join(INPUT_DIR, fname)
+        # unix_socket_path in database_global file reflects
+        # the namespace in case of multi-namespace unit-test.
+        # This is unit-test specific string to help choose the
+        # directory to load the namespace specific json files.
+        sock_path = kwargs['unix_socket_path']
```

Comment:
>unix_socket_path' [](start = 28, length = 17)

Will it throw if there is no `unix_socket_path` in `kwargs` ?

Reply:
Modified this, no longer using unix_socket_path.